nss-day-cohort-43-csharp/Fullstack-Motley-Foo
# Post Details now has a list of comments attached to it.

How to test

Login in as a user,
From the Explore Tab click on a post,
While on that post you should see any comments that are attached to that post.
Test out on multiple posts.

The comment text box DOES NOT WORK, it isn't meant to and will be redesigned in the future to work.
